Makefile

```diff
@@ -866,3 +866,49 @@ install-git-hooks:
 .PHONY: pre-commit
 pre-commit:
 	$(CONTAINERIZED) $(CALICO_BUILD) git-hooks/pre-commit-in-container
+
+CRS_VERSION := 3.3.5
+MODSEC_VERSION := v3.0.10
+
+CRS_EMBED_FOLDER := pkg/render/applicationlayer/embed/coreruleset
+CRS_ARCHIVE_SRC := https://github.com/coreruleset/coreruleset/archive/refs/tags/v$(CRS_VERSION).tar.gz
+CRS_ARCHIVE_SHA := b552005471446a4a2454a6b6ee7e65f864219ce0
+CRS_ARCHIVE_TAR := v$(CRS_VERSION).tar.gz
+CRS_ARCHIVE_TMP := $(shell mktemp -d)
+MODSEC_CONF_SRC := https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3.0.10
+MODSEC_CONF_FILES :=  modsecurity.conf-recommended unicode.mapping
+
+CHECKSUM_CMD := shasum
+
+.PHONY: generate-crs 
```

Comment:
Please make this more specific, maybe `generate-applayer-ruleset`, `CRs` are also how we refer to instances of CRDs so could be confusing.